### DISCUSSION TITLE
How to launch multiple views using CoreApplicationView and ApplicationViewSwitcher within cppwinrt application?

### DISCUSSION THREAD

**Discussion by backusd** [microsoft/WindowsAppSDK#539] [Q&A]
I have not been able to locate any documentation on how to do this using cppwinrt. I want to launch a new window using CoreApplication::CreateNewView() that runs on a different thread and therefore using something like AppWindow will not work. I've tried following the [documentation on how to do this in C#](https://docs.microsoft.com/en-us/windows/uwp/design/layout/application-view) and it simply does not work.

Also, I am doing this from an app that implements CoreApplication and uses DirectX11 (so there is no XAML in the app).

**Comment by andrewleader** [comment]
@jeffstall can you answer this?
